demo/src/data/authors-and-books/selectors/authorsAndBooks.js

```diff
@@ -0,0 +1,34 @@
+import { Selector } from "@xbyorange/mercury";
+import { booksCollection } from "../../books";
+import { authorsCollection, authorsBooksCollection } from "../../authors";
+
+// Example of a Selector with concurrent accesses to different origins and selectors
+export const authorsAndBooks = new Selector(
+  authorsCollection,
```

Comment:
Accesses to different origins are not concurrent. To achieve concurrence, sources have to be defined inside an array.